.env.dist

```diff
@@ -1,16 +1,46 @@
 export PROJECT_PATH=$(realpath .)
-export TERMINUS_SITE="terminus-test-site"
+## Note: The terminus test site must have some quicksilver scripts enabled
+## in order for tests to pass.
+
+## Needed for unit test fixutures
+export TERMINUS_PROJECT_ROOT=${PROJECT_PATH}
+
+## This is the site that will be used for testing
```

Comment:
Maybe move this comment and variable up next to the warning about quicksilver scripts above.